config/config.go

```diff
@@ -111,6 +117,140 @@ func GetNodeConfig(nodeConf *NodeConfig, cmConf *cmcfg.Config) {
 	}
 }
 
+// UpdateNodeConfigWithFlags updates NodeConfig with values from command line flags.
+func UpdateNodeConfigWithFlags(nodeConf *NodeConfig, flags *pflag.FlagSet) (err error) {
```

Comment:
As the update is done "in place" I think `*NodeConfig` should be receiver, not first argument.